lib/hyrax/linked_data_resources/geonames_resource.rb

```diff
@@ -0,0 +1,60 @@
+module Hyrax
+  module LinkedDataResources
+    # Do not Extend BaseResource For Geonames.
+    # We can get richer information from the JSON API, so let's do that instead.
+    class GeonamesResource
+      attr_reader :rdf_subject
+
+      def initialize(rdf_subject)
+        @rdf_subject = rdf_subject
+      end
+
+      # @return [String] rdf_label
+      def fetch_external
+        fetch_value
+        rdf_label
+      end
+
+      # Get the data from the URL by calling Faraday#get
+      def fetch_value
+        Rails.logger.info "Fetching #{rdf_subject} from the authorative source."
+        response = Faraday.get build_json_uri(rdf_subject)
+        return if response.status != 200
+        label(response.body)
+      rescue IOError, Faraday::ConnectionFailed, ArgumentError => e
+        # IOError could result from a 500 error on the remote server
+        # Faraday::ConnectionFailed results if there is no server to connect to
+        Rails.logger.error "Unable to fetch #{rdf_subject} from the authorative source.\n#{e.message}"
+      end
+
+      def rdf_label
+        @rdf_label ||= rdf_subject.to_s
```

Comment:
Do we need this method? Could we have `fetch_value` return the label or `rdf_subject.to_s` if there's an error?

Reply:
Yeah, I was mimicking ActiveTriples::Resource ... but that's not necessary. Done.